lib/data/devices.js

```diff
@@ -523,54 +523,6 @@ let DEVICES = {
       isLandscape: true,
     },
   },
-  'iPhone 11': {
```

Comment:
iPhone 11 entries were duplicated. This was picked up by `eslint`. Here, I'm deleting the first instances, lexically, so this should be equivalent as JS would have honored the last instance anyway.